pkg/codegen/internal/test/testdata/external-resource-schema/python/pulumi_example/workload.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import warnings
+import pulumi
+import pulumi.runtime
+from typing import Any, Mapping, Optional, Sequence, Union
+from . import _utilities, _tables
+from pulumi_kubernetes import core_v1 as _core_v1
+from pulumi_kubernetes import meta_v1 as _meta_v1
```

Comment:
Doesn't look like `_meta_v1` is being used anywhere in this file.

Reply:
This is likely related to the previous import logic as well. `meta_v1` is included transitively since it's part of the Pod schema.